### DISCUSSION TITLE
Microsoft Edge Recommends Divert to Microsoft Store

### DISCUSSION THREAD

**Discussion by GitHubRulesOK** [sumatrapdfreader/sumatrapdf#2663] [General]
@kjk 
Edge has just updated and now
Either
![image](https://user-images.githubusercontent.com/12754913/171967958-fa97cf80-40ee-40e9-bee6-629dae3a0cba.png)
OR
![image](https://user-images.githubusercontent.com/12754913/171968089-7082c3fa-e5c8-40d2-9cd3-dbe711032812.png)

**Comment by kjk** [comment]
Thanks for letting me know but I guess I can't do anything about it. Evil Microsoft be evil.
